introduction.md

```diff
@@ -1,2 +1,9 @@
 Hi, my name is Frankie Hayward and I like code.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?